### PR TITLE
typo

### PR DESCRIPTION
**Pull request by yorozen**
raw.githubuser.com/RedisLabs/redis-enterprise-k8s-docs/master/openshift/scc.yaml
should be replaced by
github.com/RedisLabs/redis-enterprise-k8s-docs/master/openshift/scc.yaml

### CODE REVIEW

File: content/kubernetes/deployment/openshift/openshift-operatorhub.md [hunk 1]
```diff
@@ -48,11 +48,11 @@ this constraint installed, the operator cannot create Redis Enterprise clusters.
 
 The security context constraint for the operator needs to be **installed only once** and **must not be deleted**.
 
-The constraint [scc.yaml](https://raw.githubuser.com/RedisLabs/redis-enterprise-k8s-docs/master/openshift/scc.yaml)
+The constraint [scc.yaml](curl -O https://github.com/RedisLabs/redis-enterprise-k8s-docs/master/openshift/scc.yaml)
```
Comment (rrelledge) [inline content/kubernetes/deployment/openshift/openshift-operatorhub.md:51]:
```suggestion
The constraint [scc.yaml](https://github.com/RedisLabs/redis-enterprise-k8s-docs/master/openshift/scc.yaml)
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
When I try to use the new links, I get a "Not Found" error message.

**Review by kaitlynmichael — COMMENTED** [review]
My subject matter experts told me that the raw url is required for curl commands. @yorozen what source are you getting this change from?

**Comment by kaitlynmichael** [comment]
Closing due to lack of activity. As far as I have been able to research, the current text is correct.

**Pull request closed without merging**